apps/styleguide/src/tables/RowEditModal.vue

```diff
@@ -108,6 +116,9 @@ export default {
     saveDraft() {
       this.executeCommand(true);
     },
+    saveEditDraft() {
+      this.executeCommand(true);
+    },
     save() {
       this.executeCommand(false);
```

Comment:
And how come you need to call this function with false? It reads to me like you shouldn't call this function at all if you don't want to executeCommand. It now reads like it has a side-effect you do want to execute?